runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -30,7 +30,7 @@ LanguageTree.__index = LanguageTree
 ---                                This is useful for overriding the built-in
 ---                                runtime file searching for the injection language
 ---                                query per language.
----@return LanguageTree |LanguageTree| parser object
+---@return LanguageTree _ |LanguageTree| parser object
```

Comment:
?

Reply:
Needed for lua-language-server to correctly parse this, otherwise it assumes `LanguageTree` is an object of type `|LanguageTree|` (with the pipes), which is breaks parsing since pipes separate multiple optional types.